lib/logstash/inputs/tcp.rb

```diff
@@ -153,7 +153,7 @@ def handle_socket(socket, client_address, client_port, output_queue, codec)
       codec.decode(read(socket)) do |event|
         event["host"] ||= client_address
         event["port"] ||= client_port
-        event["sslsubject"] ||= socket.peer_cert.subject if @ssl_enable && @ssl_verify
+        event["sslsubject"] ||= socket.peer_cert.subject.to_s if @ssl_enable && @ssl_verify
```

Comment:
I understand the problems and how this change fix it, but let me ask a few question.

Oh, So the setter on java side support what kind of objects? `String`, `Number`, `Timestamp`?
Just wondering, It cannot be done transparently without having an implicit conversion on the ruby side?